src/mgmt/cli_mgmt.c

```diff
@@ -549,6 +549,58 @@ int dc_mgmt_net_cfg(const char *name)
 	return rc;
 }
 
+/*
+ * Send an upcall to the agent to notify it of a clean process shutdown.
+ */
+int
+dc_mgmt_disconnect(void)
+{
+	struct drpc_alloc	 alloc = PROTO_ALLOCATOR_INIT(alloc);
+	struct drpc		*ctx;
+	Drpc__Call		*dreq;
+	Drpc__Response		*dresp;
+	int			rc;
+
+	D_DEBUG(DB_MGMT, "disconnecting process for pid:%d\n", getpid());
+
+	/* Connect to daos_agent. */
+	D_ASSERT(dc_agent_sockpath != NULL);
+	rc = drpc_connect(dc_agent_sockpath, &ctx);
+	if (rc != -DER_SUCCESS) {
+		D_ERROR("failed to connect to %s " DF_RC "\n",
+			dc_agent_sockpath, DP_RC(rc));
+		D_GOTO(out, 0);
+	}
+
+	rc = drpc_call_create(ctx, DRPC_MODULE_MGMT,
+			      DRPC_METHOD_MGMT_DISCONNECT, &dreq);
+	if (rc != 0) {
+		goto out_ctx;
+	}
```

Comment:
the tiniest nit - DAOS C coding style has been not using the curly brackets when the if-block is a single statement

Reply:
Fixed